Tests/TemplateKitTests/TemplateDataEncoderTests.swift

```diff
@@ -59,6 +59,14 @@ class TemplateDataEncoderTests: XCTestCase {
             "hello": .string("hello"),
         ]))
     }
+    
+    func testEncodableWithURL() {
```

Comment:
This test needs to be added to the `allTests` variable (bottom or top of the file) to be run on Linux.

Reply:
Yes, I'm new to this so I always forget it

Added